feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/captive.c

```diff
@@ -368,15 +428,37 @@ void vif_captive_portal_set(const struct schema_Wifi_VIF_Config *vconf, char *if
 
 		if (!strcmp(opt, "authentication")) {
 			//blobmsg_add_u32(&cap, "gatewayinterface",ifname);
-			//if(strcmp(value,"None")==0) {
-			blobmsg_add_string(&cap, "authentication", value);
-			blobmsg_add_string(&cap, "login_option_enabled", "1");
-			blobmsg_add_string(&cap, "enabled", "1");
 			blobmsg_add_string(&cap, "webroot",webroot);
-			//}
-			/*else if(strcmp(value,"Captive Portal User List")==0) {
+			if(strcmp(value,"None")==0) {
```

Comment:
Can we follow Kernel coding style guideline?  Please refer section 3 of the document (Placing braces and whitespaces)